sdk/storage/Azure.Storage.Queues/samples/Sample01a_HelloWorld.cs

```diff
@@ -184,7 +184,7 @@ public static void IdentityAuth()
             #region Snippet:Azure_Storage_Queues_Samples_Sample01a_HelloWorld_IdentityAuth
 
             // Create a QueueClient that will authenticate through Active Directory
-            Uri accountUri = new Uri("https://MYSTORAGEACCOUNT.blob.core.windows.net/");
+            Uri accountUri = new Uri("https://MYSTORAGEACCOUNT.queue.core.windows.net/QUEUENAME");
```

Comment:
could you please change `accountUri` to `queueUri` ?

Reply:
hey Kamil, i have done these changes. Thanks.